src/Workspaces/Remote/Core/Services/AssetService.cs

```diff
@@ -17,39 +17,70 @@ namespace Microsoft.CodeAnalysis.Remote
     /// </summary>
     internal class AssetService
     {
+        private const int CleanupInterval = 30000; // 30 seconds
```

Comment:
Yeah, I just guessed at the time values. CleanupInterval is how often the service wakes up and checks the cache, then PurgeAfter is how long before an asset is purged. (Yes, the code got messed up when I introduced the constant, I'll fix that.) What values do you think we should use? Check for cleanup every 3 minutes and purge assets after 5 minutes?